.eslintrc.js

```diff
@@ -1,8 +1,37 @@
+const LICENSE_HEADER = `
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+`
```

Comment:
Is this ` expected?

Reply:
the ` works as multiline quotes, to make the license header a multi line string to lint when checking

Reply:
Thanks,?I didn’t catch that